README.md

```diff
@@ -1,107 +1,106 @@
 
 - If everything is deployed correctly, your document should start uploading to your cluster under the `chatter > training_data` collection.
 
-- Your data should now start appearing as below in the collection.
+- Your data should appear like this in the collection:
 
   ![image](https://github.com/utsavMongoDB/MongoDB-RAG-NextJS/assets/114057324/316af753-8f7b-492f-b51a-c23c109a3fac)
 
 
+### Step 5: Create Vector Index on Atlas
+For the RAG Question Answering (QnA) to work, you need to create a Vector Search Index on Atlas so your vector data can be fetched and served to LLMs.
 
-#### Step 4: Create Vector Index on Atlas
-- Now for the RAG (QnA) to work, you need to create a Vector Search Index on Atlas so the vector data can be fetched and served to LLMs.
+Let’s head over to our MongoDB Atlas user interface to create our Vector Search Index.
 
-  #### Create a search index as below.
-
--  Let’s head over to our MongoDB Atlas user interface to create our Vector Search Index. First, click on the “Search” tab and then on “Create Search Index.” You’ll be taken to this page (shown below). Please click on “JSON Editor.”
+- First, click on "Atlas Search” in the sidebar of the Atlas dashboard. Select the cluster you're using for this guide. Then click “Create Search Index.” 
+- You’ll be taken to this page (shown below). Here, select “JSON Editor” in the Atlas Vector Search section. Click "Next".
     ![image](https://github.com/utsavMongoDB/MongoDB-RAG-NextJS/assets/114057324/b41a09a8-9875-4e5d-9549-e62652389d33)
 
-- Next, input the values shown in the image below and create the Vector.
-
-  ````
-    {
-      ields": [
-        {
-          "type": "vector",
-          "path": "text_embedding",
-          "numDimensions": 1536,
-          "similarity": "cosine",
-        }
-      ]
-    }
-  ````
+- Input the values shown below and create the vector index.
+    ````
+      {
+        "fields": [
+          {
+            "type": "vector",
+            "path": "text_embedding",
+            "numDimensions": 1536,
+            "similarity": "cosine"
+          }
+        ]
+      }
+    ````
 
   ![image](https://github.com/utsavMongoDB/MongoDB-RAG-NextJS/assets/114057324/d7e560b3-695c-4210-8a6d-ea50c589bc70)
 
 - You should start seeing a vector index getting created. You should get an email once index creation is completed.
   ![image](https://github.com/utsavMongoDB/MongoDB-RAG-NextJS/assets/114057324/c1842069-4080-4251-8269-08d9398e09aa)
 
 
-#### Step 5: Ask questions
-- Once completed, head to the QnA section to start asking questions based on your trained data, and you should get the desired response.
+### Step 6: Ask questions
+Finally, head back to your chatbot website. Select the "QnA" tab to start asking questions based on your trained data.
 
   ![image](https://github.com/utsavMongoDB/MongoDB-RAG-NextJS/assets/114057324/c76c8c19-e18a-46b1-834a-9a6bda7fec99)
 
 
 
-
-### Option 2
```

Comment:
Removing this because it's likely to lead to confusion: The reader still needs to apply steps from the "Option 1" path.